edg/parts/Ina219.py

```diff
@@ -0,0 +1,77 @@
+from edg import *
+from edg.parts.JlcPart import JlcPart
+
+
+class Ina219_Device(InternalSubcircuit, JlcPart, FootprintBlock, Block):
+    @init_in_parent
+    def __init__(self):
+        super().__init__()
+
+        self.vs = self.Port(VoltageSink(
+            voltage_limits=(3, 5.5) * Volt,
+            current_draw=(6 * uAmp, 1 * mAmp)
+        ))
+        self.gnd = self.Port(Ground())
+
+        dio_model = DigitalBidir.from_supply(
+            self.gnd, self.vs,
+            voltage_limit_tolerance=(-0.3, 0.3) * Volt,
+            input_threshold_factor=(0.3, 0.7)
+        )
+        dio_sda_model = DigitalBidir.from_supply(
+            self.gnd, self.vs,
+            voltage_limit_abs=(-0.3 * Volt, 6.0 * Volt),
+            input_threshold_factor=(0.3, 0.7)
+        )
+
+        self.i2c = self.Port(I2cTarget(DigitalBidir.empty(), addresses=[0x40]))
+        self.i2c.sda.init_from(dio_sda_model)
+        self.i2c.scl.init_from(DigitalSink.from_bidir(dio_model))
+
+        self.in_pos = self.Port(AnalogSink(voltage_limits=(0.0, 26) * Volt))
+        self.in_neg = self.Port(AnalogSink(voltage_limits=(0.0, 26) * Volt))
+
+    def contents(self):
+        super().contents()
+        self.footprint(
+            'U', 'Package_TO_SOT_SMD:SOT-23-8',
+            {
+                '1': self.in_pos,
+                '2': self.in_neg,
+                '3': self.gnd,
+                '4': self.vs,
+                '5': self.i2c.scl,
+                '6': self.i2c.sda,
+                '7': self.gnd,  # TODO: make this address selectable, sa0 pin
+                '8': self.gnd  # TODO: make this address selectable, sa1 pin
+            },
+            mfr='Texas Instruments', part='INA219AIDCNR',
+            datasheet='https://www.ti.com/lit/ds/symlink/ina219.pdf'
+        )
+        self.assign(self.lcsc_part, "C87469")
```

Comment:
```suggestion
        self.assign(self.lcsc_part, "C87469")
        self.assign(self.actual_basic_part, False)
```
Should also add that it's not a basic part instead of leaving it unassigned